roles/openshift_management/README.md

```diff
@@ -268,6 +254,9 @@ openshift_management_app_template=cfme-template-ext-db
 openshift_management_template_parameters={'DATABASE_USER': 'root', 'DATABASE_PASSWORD': 'r1ck&M0r7y', 'DATABASE_IP': '10.10.10.10', 'DATABASE_PORT': '5432', 'DATABASE_NAME': 'cfme'}
 ```
 
+**NOTE:** Ensure your are running PostgreSQL 9.5 or you may not be
```

Comment:
Does this mean > 9.6 won't work? Or that it must be at least 9.5?

Reply:
Meaning I only know that 9.5 is tested and that is the version of postgres they told me they intentionally bundle in their `manageiq-pods` images and in their appliance.

Reply:
OK, so it is as it reads and means use only 9.5. Just wanted to make sure.

Reply:
NIT: 'your are' -> 'you are'